invoice/views.py

```diff
@@ -181,11 +181,15 @@ def process_job(self):
 
 		rt = RTResource(settings.RT_URL, settings.RT_USER, settings.RT_PW, BasicAuthenticator)
 		# TODO: Refactor froscon specific parts to config
```

Comment:
Yes, I would. I have no experience with Django though and would need some hints to point me in the right direction. Where should this be configured?

It would help if you could merge the change as it is for now, so that future invoices use the new subject.